### ISSUE TITLE
Issue with 'play_fav_radio_station_no' function

### ISSUE THREAD

**Issue by Bruno21** [avantrec/soco-cli#12]
Hi,

play_fav_radio_station_no  don't always play the right radio:

```
sonos -l Salon favourite_radio_stations

    1: Amazing Blues
    2: France Inter 95.9 (Émissions-débats France)
    3: K6 FM 101.6 (Musique Locale)
    4: RTL 104.2 (Monde)
    5: Radio Classique 105.8 (Classique)
    6: Rire et Chansons 91.8 (Comédie)
    7: SomaFM: Indie Pop Rocks!
    8: franceinfo 101.2 (Émissions-débats France)

```

```
sonos -l Salon play_fav_radio_station_no 1    # play 1 - OK
sonos -l Salon play_fav_radio_station_no 3    # play 8 - NOK

```

play_fav_radio_station_no 1 -> play 1
play_fav_radio_station_no 2 -> play 2
play_fav_radio_station_no 3 -> play 8
play_fav_radio_station_no 4 -> play 3
play_fav_radio_station_no 5 -> play 5
play_fav_radio_station_no 6 -> play 6
play_fav_radio_station_no 7 -> play 4
play_fav_radio_station_no 8 -> play 7

**Comment by pwt** [comment]
Thanks. Looks like a sorting bug. I'll take a look later.

**Comment by pwt** [comment]
Should be fixed in v0.3.12.

**Comment by Bruno21** [comment]
Perfect !
